frontend/src/Form/beta/JsonSchemaForm.tsx

```diff
@@ -58,6 +58,26 @@ function DescriptionFieldTemplate() {
   return <></>
 }
 
+function ObjectFieldTemplate(props: ObjectFieldTemplateProps) {
```

Comment:
Can we destructure these props so we don't have to access them via `props.` below?